### ISSUE TITLE
Rebuild `setuptools` 65 

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/miniforge#409]
As discussed in issue ( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/3973 ), `conda-build` fails with `setuptools` 66. A hot-fix for `conda-build` was added in PR ( https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/387 ). Though it looks like the installer may already ship with `setuptools` 66. Think we may need to rebuild it with `setuptools` pinned to 65 to avoid issues with `conda-build`. Though users could always update to a newer `setuptools` if they prefer

**Comment by hmaarrfk** [comment]
The last build log on main shows version 65

https://github.com/conda-forge/miniforge/actions/runs/3766197110/jobs/6402449244#step:4:322

**Comment by jakirkham** [comment]
Yeah think you are right. Yesterday we had seen 66 show up somewhere and then get downgraded. So thought maybe it was included in the installer, but the end think something else was going on

Maybe we can leave this open as a reminder? At least until we've figured out a better solution

**Comment by hmaarrfk** [comment]
yes agreed.

**Comment by jakirkham** [comment]
How do you think we should handle this at this point?

**Comment by hmaarrfk** [comment]
i'm not sure what the status is on compatibility. I've been pinning it in the installer.

**Comment by hmaarrfk** [comment]
users are free to update it. the installer just needs to be "stable"

**Comment by jakirkham** [comment]
No worries. Pinning in the installer makes sense. Agree users can update.

In terms of fixing the issue in conda-build, PR ( https://github.com/conda/conda-build/pull/4443 ) should do that. Waiting for feedback atm.
